Web_Development/Javascript/7.The DOM/DOM - Examples/Example1 - Random BG color generator/randomColor.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@500&display=swap');
+
+.content{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    font-family: 'Quicksand', sans-serif; 
+    margin: auto;
+    position: absolute;
+    top: 0; left: 0; bottom: 0; right: 0;
+    
+}
+
+.box{
+    background-color: white;
+    border: 2px solid black;
+    border-radius: 25px;
+    padding-left: 10px;
+    padding-right: 10px;
+    font-size: 30px;
+    font-weight: 600;
+}
+
+.changebtn{
+    margin-top: 10px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    padding-left: 20px;
+    padding-right: 20px;
+    background-color: white;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 20px;
+    border-radius: 10px;
+}
+
+.changebtn:hover{
+    background-color: black;
+    color: white;
+}
+
+/* footer */
+
+.footer {
+    font-family: 'Quicksand', sans-serif;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    background-color: rgba(255, 255, 255, 0.253);
+    color: black;
+    text-align: center;
+  }
+
+.heart{
+    color: red;
+}
+
+.github{
+    text-decoration: none;
+}
+
+.gwoc{
+    color: tomato;
+}
```

Comment:
Let's keep to markdown files not code files.
Make sure the code is indented properly.

Reply:
Required changes made in latest commit.